mp/src/game/client/c_baseplayer.cpp

```diff
@@ -1263,11 +1263,11 @@ void C_BasePlayer::Flashlight( void )
 	UpdateFlashlight();
 
 	// Check for muzzle flash and apply to view model
-	C_BaseAnimating *ve = this;
+	/*C_BaseAnimating *ve = this;
```

Comment:
Why was this commented?

Reply:
It was doing a dynamic cast every tick as a spectator, especially when in first person, for no reason.